pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2183,37 +2183,33 @@ func buildListener(opts buildListenerOpts) *xdsapi.Listener {
 // This allows external https traffic, even when port the port (usually 443) is in use by another service.
 func appendListenerFallthroughRoute(l *xdsapi.Listener, opts *buildListenerOpts,
 	node *model.Proxy, currentListenerEntry *outboundListenerEntry) {
-	if features.EnableFallthroughRoute.Get() {
+	wildcardMatch := &listener.FilterChainMatch{}
```

Comment:
not changed in this PR, but can you move this to line 2212 where we append to filterchain so that we only construct it when needed?

Reply:
sure